scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableSyntaxConfig.scala

```diff
@@ -7,21 +7,21 @@ import MetaconfigPendingUpstream.XtensionConfScalafix
 import scala.meta.tokens.Token
 
 case class DisableSyntaxConfig(
-    carriageReturn: Boolean = false,
+    noCarriageReturn: Boolean = false,
     keywords: Set[DisabledKeyword] = Set(),
-    semicolons: Boolean = false,
-    tabs: Boolean = false,
-    xml: Boolean = false
+    noSemicolons: Boolean = false,
+    noTabs: Boolean = false,
+    noXml: Boolean = false
 ) {
   implicit val reader: ConfDecoder[DisableSyntaxConfig] =
     ConfDecoder.instanceF[DisableSyntaxConfig](
       c =>
         (
-          c.getField(carriageReturn) |@|
+          c.getField(noCarriageReturn) |@|
```

Comment:
Did no tests fail from this change?

Reply:
Adding a test for this.

Reply:
I think we should remove this one. I modified the file with an hex editor to add a <CR> but it was not picked up by scalameta.